src/main/java/teammates/ui/controller/StudentFeedbackResultsPageData.java

```diff
@@ -105,8 +105,15 @@ private FeedbackResultsQuestionDetails createQuestionDetails(
         
         String questionText = questionDetailsBundle.questionText;
         String additionalInfo = questionDetailsBundle.getQuestionAdditionalInfoHtml(questionIndex, "");
+        String studentEmail;
```

Comment:
should be better if you test for null rather than catch a null pointer exception?